app/javascript/src/Plans/components/DefaultPlanSelectCard.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React, { useState } from 'react'
+import { post } from 'utilities/ajax'
+
+import {
+  Form,
+  FormGroup,
+  Card,
+  CardBody
+} from '@patternfly/react-core'
+import { DefaultPlanSelect } from 'Plans'
+import { Spinner } from 'Common'
+
+import type { Product, ApplicationPlan } from 'Types'
+
+import './DefaultPlanSelectCard.scss'
+
```

Comment:
I'd rather omit empty lines, and group all imports as a block

Reply:
Everything for an approval 🙏 🤣